templates/misc/nav-toggle-button.twig

```diff
@@ -10,14 +10,14 @@
 
 <div class="nav-toggle {{modifier_class}}">
   {# Fallback menu button #}
-  <div class="nav-toggle__anchor">
-    <a href="#" class="nav-toggle__label nav-toggle__label--close">{{ menu_close }}</a>
-    <a href={{ anchor_target }} class="nav-toggle__label nav-toggle__label--open">{{ menu_open }}</a>
+  <div class="nav-toggle__anchor{% if modifier_suffix %} nav-toggle__anchor--{{ modifier_suffix }}{% endif %}">
```

Comment:
What is this `modifier_suffix`? I think we should try to avoid creating classes with variables like this. ( I know it's annoying to avoid) But it's really hard to search for `nav-toggle__anchor--something` when the string is not there. Also it's hard for me to see from this what values are supported.

Could we do an (annoying) if-else-then in twig that toggles between these on this same twig file? So that we'd have fully written supported classes on this file?

We could also avoid the four if-then statements as we could just set the `modifier_class` within that if-else-then statement.

Reply:
The modifier suffix is there to tackle the issue here: https://github.com/City-of-Helsinki/drupal-hdbt/pull/523#discussion_r1087652218 so there is no way I can create a BEM-modifier class without using a variable here I think. I don't think I quite understand what you mean with the if-else-then in this case?

Reply:
I mean the approach for creating the class like in this file: templates/misc/status-label.html.twig instead of what is used here.

Reply:
Aaa - that would mean we need to hard-code the modifier options to the template. I don't know if that is something that I would sacrifice over the ease of search with the modifier class at this point.

Reply:
Changed this as we discussed.

Reply:
Big thanks for listening. I think it's way cleaner now as we all can now see what modifiers are supposed to be supported from the template. Thank you!